publish_buf.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/sh
-
-set -euxo pipefail
-
-ITER=${1:-1}
-
-buildImage() {
-    PLUGIN=$1
-    IMAGE=$2
-    VERSION=$(cargo read-manifest --manifest-path "protoc-gen-$PLUGIN/Cargo.toml" 2>/dev/null | jq -r .version)
-
-    docker buildx build --platform=linux/amd64 --build-arg "BIN=protoc-gen-$PLUGIN" -t "plugins.buf.build/prost/$IMAGE:$VERSION-$ITER" --push .
```

Comment:
FYI the `plugins.buf.build` domain is going away with the alpha remote removal. This script will no longer be functional so it should be safe to remove.